lib/shark_client.js

```diff
@@ -129,6 +128,26 @@ function _request(opts, cb) {
     }
 }
 
+/*
+ * Use information about the object to craft the storage path.
+ */
+function storagePath(opts) {
+    if (opts.storageLayoutVersion === 2) {
+        var storageLayoutVersion = '/v' + opts.storageLayoutVersion + '/';
+        var owner = opts.owner;
+        var bucketId = opts.bucketId;
+        var objectNameHash = opts.objectNameHash;
+        var objectId = opts.objectId;
+        var objectIdAndHash = objectId + ',' + objectNameHash;
+        var objectIdPrefix = objectId.substr(0, 2);
```

Comment:
Perhaps pull this `2` out to a constant? Or at the least a comment here or on this function for why "2" here.